build.sh

```diff
@@ -299,8 +299,9 @@ sudo apt-get -y update &>> ${CWD}/netflix-proxy.log\
   && sudo apt-get -y install git python3.6 python3-venv python3-pip sqlite3 &>> ${CWD}/netflix-proxy.log\
   && python3 -m venv venv &>> ${CWD}/netflix-proxy.log\
   && source venv/bin/activate &>> ${CWD}/netflix-proxy.log\
-  && pip3 install -r requirements.txt &>> ${CWD}/netflix-proxy.log\
```

Comment:
@nishuo 

could you please add pip upgrade before the installation of requirements as follows:

```
...
 && source venv/bin/activate &>> ${CWD}/netflix-proxy.log\
+  && pip3 install pip --upgrade &>> ${CWD}/netflix-proxy.log\
  && pip3 install -r requirements.txt &>> ${CWD}/netflix-proxy.log\
```

This is the only change that is required.
```